lib/brains/management/commands/samples/webhooks.py

```diff
@@ -193,3 +193,58 @@
   </subject>
 </notification>
 """  # noqa
+
+no_trans = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
mmm, more XML. This will be not fun to maintain. I can't offer a better idea though.